backend/data/src/main/kotlin/io/tolgee/service/dataImport/processors/FileProcessorContext.kt

```diff
@@ -90,9 +90,7 @@ data class FileProcessorContext(
 
       val key = getOrCreateKey(keyName)
```

Comment:
Can you please remove this whole part and remove the metadata field completely from here? Then call the description adding from directly form your processor.